src/utils/AppRouteProvider.jsx

```diff
@@ -37,6 +48,19 @@ const AppRouteProvider = (props) => {
       setRenderIntercept(availableIntercepts.DATA_PROCESSING(nextRoute, hardNavigate));
       return;
     }
+
+    if (previousRoute.match('/data-management')) {
+      const lastViewed = moment().toISOString();
+
+      dispatch(updateExperiment(experimentId, { lastViewed }));
+      dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
```

Comment:
this is out of the scope of this PR, but these two lines do not make sense:
```
dispatch(updateExperiment(experimentId, { lastViewed }));
dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
```
Having to make 2 extra requests with "modify permission" (PUT) to update `lastViewed` is completely unnecessary. This should be a side effect in the API endpoints. ie., whenever the endpoint corresponding to `updateExperimentInfo` is hit, then set those two variables from the API (possibly with a hook).

Reply:
Should I make a ticket for this? This will be irrelevant with the SQL changes.

Reply:
yes that would be great. although it won't be irrelevant with SQL changes. SQL can keep track of insertion date easily, tracking access is more complicated and probably not worth it (easier to just update modification date from the API code when doing the update itself).

Reply:
are we actually using both values `lastViewed` and `lastAnalyzed` for something?  or are we just using one a duplicating the info

Reply:
Previously it would give an idea of when a project was last opened, and when an experiment within a project was last viewed (last analyzed). The idea was you may have multiple experiments per project, and you can see the last viewed date of each experiment.

Reply:
So by removing projects, we don't need `lastAnalyzed` on project

Reply:
I will remove them in another PR as agreed

Reply:
Feedback from Vicky during standup on the 18th of Jan 2022 : We can delete last analyzed